examples/low-level-api/private_tangle.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! A basic example that generates and publishes a DID Document
+//! to a private tangle.
+//! It can be run together with a local hornet node.
+//! Refer to https://github.com/iotaledger/one-click-tangle/tree/chrysalis/hornet-private-net
+//! for setup instructions.
+//!
+//! cargo run --example private_tangle
+
+use identity::iota::ClientBuilder;
+use identity::iota::ClientMap;
+use identity::iota::Network;
+use identity::iota::Receipt;
+use identity::prelude::*;
+
+#[tokio::main]
+pub async fn main() -> Result<()> {
+  // This is an arbitrarily defined network name
+  let network_name = "atoi";
+
+  // Unwrap is fine since we provided a non-empty string.
+  let network = Network::from_name(network_name)?;
```

Comment:
Standardise example network names to `"custom"`.

```suggestion
  // This is an arbitrarily defined network name
  let network_name = "custom";
  let network = Network::from_name(network_name)?;
```